typings/telegram.d.ts

```diff
@@ -0,0 +1,660 @@
+import * as tt from './telegram-types.d'
+
+import * as https from 'https'
+import * as http from 'http'
+
+interface AdminPerms {
```

Comment:
I think interfaces like this should be exported in order to be declared before used with types

```ts
const param: AdminPerms = {…}
ctx.method(…, params, …)
```